RoundTimer.py

```diff
@@ -17,16 +18,19 @@
 logging.basicConfig(level=logging.INFO, format='[%(levelname)s] (%(threadName)s) %(message)s')
 
 # Set up threads and matrix output
-threads = []
 matrix = Adafruit_RGBmatrix(32, 2)
 
 def kill(signal, frame):
     '''
     Trap the signal and kill any ongoing threads, then exit
     '''
     for thread in threads:
+        try:
+            thread.pause.set()
+        except:
+            pass
```

Comment:
only some threads have pause we just want to try to do all of those and dont care about the others but i'll add comment